src/components/ProductTours/data/hooks.js

```diff
@@ -29,14 +29,14 @@ export const useBrowseAndRequestTour = () => {
 };
 
 export const useLearnerCreditTour = () => {
-  const history = useHistory();
-  const inLearnerCreditPage = history.location.pathname.includes(ROUTE_NAMES.learnerCredit);
+  const { params: { page } } = useRouteMatch();
```

Comment:
nit: could use `useParams` here if we're only using the params from `useRouteMatch`